cst438-homework2/src/main/java/cst438/service/CityService.java

```diff
@@ -19,12 +21,19 @@
   CountryRepository countryRepository;
   @Autowired
   WeatherService weatherService;
+  @Autowired
+  private RabbitTemplate rabbitTemplate;
+  @Autowired
+  private FanoutExchange fanout;
+
 
   public CityService(CityRepository cityRepository, CountryRepository countryRepository,
-      WeatherService weatherService) {
+      WeatherService weatherService, RabbitTemplate rabbitTemplate, FanoutExchange fanout) {
```

Comment:
This is great! This would have solved my problems when I tried to write my tests!